godwin.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html><head>
+<meta http-equiv="content-type" content="text/html; charset=windows-1252">
+		<title>HTML Attributes</title>
+	</head>
+	<body>
+		<h1 title="Lesson 3" style="color: red; text-align: center;">Attributes</h1>
+		<p id="firstParagraph" title="Attributes" style="color: red;">Welcome to HTML</p><hr>
```

Comment:
Why do you have `<hr>` here? Please check the proper way of how to use an empty element. Thanks.